src/primerProblema.js

```diff
@@ -0,0 +1,48 @@
+//Reto: ENCONTAR ERRORES
+
+//var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+
+//Se agregaron console.log para depurar el código e ir encontrado los errores
+
+var API = 'https://rickandmortyapi.com/api/character/';
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```